pvlib/test/test_solarposition.py

```diff
@@ -134,49 +185,140 @@ def test_get_sun_rise_set_transit():
+    frame = pd.DataFrame({'sunrise': sunrise, 'sunset': sunset}, index=times)
+
     result = solarposition.get_sun_rise_set_transit(times, south.latitude,
                                                     south.longitude,
-                                                    delta_t=64.0)
-    frame = pd.DataFrame({'sunrise':sunrise, 'sunset':sunset}, index=times)
+                                                    delta_t=65.0)
     result_rounded = pd.DataFrame(index=result.index)
     # need to iterate because to_datetime does not accept 2D data
     # the rounding fails on pandas < 0.17
     for col, data in result.iteritems():
-        result_rounded[col] = pd.to_datetime(
-            np.floor(data.values.astype(np.int64) / 1e9)*1e9, utc=True)
+        result_rounded[col] = data.dt.round('1s')
 
     del result_rounded['transit']
     assert_frame_equal(frame, result_rounded)
 
+    # test for Golden, CO compare to NREL SPA
+    result = solarposition.get_sun_rise_set_transit(
+        expected_rise_set_spa.index, golden.latitude, golden.longitude,
+        delta_t=65.0)
 
-    # tests from USNO
-    # Golden
-    golden = Location(39.0, -105.0, tz='MST')
-    times = pd.DatetimeIndex([datetime.datetime(2015, 1, 2),
-                              datetime.datetime(2015, 8, 2),]
-                             ).tz_localize('MST')
-    sunrise = pd.DatetimeIndex([datetime.datetime(2015, 1, 2, 7, 19, 2),
-                                datetime.datetime(2015, 8, 2, 5, 1, 26)
-                                ]).tz_localize('MST').tolist()
-    sunset = pd.DatetimeIndex([datetime.datetime(2015, 1, 2, 16, 49, 10),
-                               datetime.datetime(2015, 8, 2, 19, 11, 31)
-                               ]).tz_localize('MST').tolist()
-    result = solarposition.get_sun_rise_set_transit(times, golden.latitude,
-                                                    golden.longitude,
-                                                    delta_t=64.0)
-    frame = pd.DataFrame({'sunrise':sunrise, 'sunset':sunset}, index=times)
+    # round to nearest minute
     result_rounded = pd.DataFrame(index=result.index)
     # need to iterate because to_datetime does not accept 2D data
-    # the rounding fails on pandas < 0.17
     for col, data in result.iteritems():
-        result_rounded[col] = (pd.to_datetime(
-            np.floor(data.values.astype(np.int64) / 1e9)*1e9, utc=True)
-            .tz_convert('MST'))
+        result_rounded[col] = data.dt.round('s').tz_convert('MST')
 
-    del result_rounded['transit']
-    assert_frame_equal(frame, result_rounded)
+    assert_frame_equal(expected_rise_set_spa, result_rounded)
+
+
+@requires_ephem
+def test_get_rise_set_ephem(expected_rise_set_ephem):
+    # test for Golden, CO compare to USNO, using local midnight
+    result = solarposition.get_rise_set_ephem(expected_rise_set_ephem.index,
+                                              golden.latitude,
```

Comment:
IMO `golden` is unexpected here, can you please use caps so future maintainers will know that `GOLDEN` is a constant defined at the top of the module? This will also require it to be changed on [line 26](https://github.com/pvlib/pvlib-python/blob/36180223fc819106b11dd8fae8f9e898094c7479/pvlib/test/test_solarposition.py#L26) as well